signer/core/apitypes/types.go

```diff
@@ -251,6 +251,24 @@ type TypedDataDomain struct {
 	Salt              string                `json:"salt"`
 }
 
+func TypedDataHash(typedData TypedData) ([]byte, error) {
+	hash, _, err := TypedDataAndHash(typedData)
+	return hash, err
+}
+
```

Comment:
This is unused, right? Let's not add it, it's trivial for a caller to ignore the second arg

Reply:
I added it to mirror the construction of the existing `TextHash` and `TextAndHash` methods. I'd agree it's kind of pointless though, and will remove it.